kokoro/release/python/linux/build_artifacts.sh

```diff
@@ -30,7 +30,7 @@ git clone https://github.com/matthew-brett/multibuild.git
 # silently creeping in (see https://github.com/protocolbuffers/protobuf/issues/9180).
 # IMPORTANT: always pin multibuild at the same commit for:
 # - linux/build_artifacts.sh
-# - linux/build_artifacts.sh
+# - macos/build_artifacts.sh
```

Comment:
Were these changes meant to be included? I think this PR must have been based on the 3.20.x branch because it seems to be including some commits that are on 3.20.x but haven't been merged to main yet.

Reply:
No this was based on main.  I just came across these and got confused, and thought I should update them

Reply:
Ah ok, I see. It's still strange that a bunch of unrelated commits show up here: https://github.com/protocolbuffers/protobuf/pull/9793/commits But I guess if you do "squash and merge" then it won't matter.

Reply:
Ah yea I think I was originally off of 3.20.x and switched to main in commit 72c6de816143fcae34e24c4a2b1ae3d8b70b05de.  Squash and merge should keep it hidden though :)